presto-docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -182,6 +182,7 @@ Elasticsearch Presto
 ``keyword``   ``VARCHAR``
 ``text``      ``VARCHAR``
 ``date``      ``TIMESTAMP``
+``ip``        ``VARCHAR``
```

Comment:
see https://prestosql.io/docs/current/language/types.html#ipaddress

Reply:
Agreed, it would be better to map this to `ipaddress`

Reply:
I change it.. but I'm unable to make this work..
the code to read the value is on IpAddressDecoder

can someone guide me?

Reply:
`IPADDRESS.writeSlice(blockBuilder, slice);` -> `IPADDRESS.writeSlice(output, slice);`

you don't need intermediate block build, see 
https://github.com/prestosql/presto/blob/677df2ced0306609d870c75b75da347c4a688e62/presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/decoders/VarbinaryDecoder.java#L37-L39